packages/kit/src/runtime/client/client.js

```diff
@@ -223,6 +223,13 @@ export async function start(_app, _target, hydrate) {
 		);
 	}
 
+	// detect basic auth credentials in the current URL
+	// https://github.com/sveltejs/kit/pull/11179
+	// if so, refresh the page without credentials
+	if (document.URL !== location.href) {
+		location.href = `${location.href}`;
```

Comment:
no need to coerce this

```suggestion
		location.href = location.href;
```

Reply:
(though we might need an eslint comment... bah)

Reply:
yeah that was why I did that...